cmd/template/main.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Intel Corporation. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/intel/nri-resmgr/pkg/config"
+	"github.com/intel/nri-resmgr/pkg/instrumentation"
+	"github.com/intel/nri-resmgr/pkg/resmgr"
+
+	logger "github.com/intel/nri-resmgr/pkg/log"
+	version "github.com/intel/nri-resmgr/pkg/version"
+
+	_ "github.com/intel/nri-resmgr/cmd/template/policy"
```

Comment:
The module location has changed, please change
`intel/nri-resmgr/`
to
`containers/nri-plugins`